### ISSUE TITLE
No error reported with "ZERO_VECTOR.x" and similar constructions

### ISSUE THREAD

**Issue by buildersbrewery** [Makopo/lslint#63]
The following code does **NOT** compile in Second Life:

> Error should be at (5, 30) at the '.'

```lsl
default
{
    state_entry()
    {
        float f = ZERO_VECTOR.x;
        llOwnerSay((string)f);
    }
}
```

however lslint does not complain. Tested with `v1.0.8`.

**Comment by Sei-Lisa** [comment]
Thanks for the report. It's a consequence of commit 24127a6 (the one that introduces builtins.txt), and it's irritating, because in that commit, constants are treated by the syntax analyzer like any other variables, making it more difficult to catch that situation.

**Comment by Sei-Lisa** [comment]
#64 should fix this, though I'd appreciate feedback regarding the error message.
